odl_video/settings.py

```diff
@@ -104,6 +104,9 @@ def get_var(name, default):
 # Application definition
 
 INSTALLED_APPS = [
+    'ui.apps.UIConfig',
+    'cloudsync.apps.CloudSyncConfig',
+    'dj_elastictranscoder',
```

Comment:
can you please explain me why the change of order and if you see any side effect by doing this?